plugins/coral-plugin-offtopic/client/containers/OffTopicFilter.js

```diff
@@ -0,0 +1,27 @@
+import {connect} from 'react-redux';
+import {bindActionCreators} from 'redux';
+import {toggleCheckbox} from '../actions';
+import {
+  addClassName,
+  removeClassName
+} from 'coral-embed-stream/src/actions/comment';
+import OffTopicFilter from '../components/OffTopicFilter';
+import {closeViewingOptions} from 'coral-embed-stream/src/actions/stream';
```

Comment:
We need to provide this actions through the `coral-plugin-api` in a folder called `alpha` or `beta` if we are more confident that it doesn't change.